src/nn/wasm/src/bind/src/binding.cpp

```diff
@@ -293,6 +336,12 @@ EMSCRIPTEN_BINDINGS(nn)
     .field("float_activation_max", &ArithmeticParams::float_activation_max)
     ;
 
+  class_<TransposeParams>("TransposeParams")
```

Comment:
Here expose `TransposeParams` as a class, because `TransposeParams` have a property `perm` which is a int array. And it can't be exposed by simple `value_object`.

Reply:
`value_object` should support array field. Please refer to https://github.com/emscripten-core/emscripten/blob/incoming/tests/embind/embind_test.cpp#L1781

Reply:
Any updates on this? Is it possible to use this method?